product_state/models/product_template.py

```diff
@@ -10,6 +10,7 @@ class ProductState(models.Model):
     _order = 'name'
 
     name = fields.Char('State Name', required=True)
+    state = fields.Char(related=name)
```

Comment:
@rousseldenis You mean this?

Reply:
Nope. Here add something like :

```python
code = fields.Char()
```
to store the former Selection() state value.

And in product :

```python
state = fields.Char(related=product_state_id.code)
```